packages/js/src/generators/library/library.ts

```diff
@@ -451,7 +451,8 @@ function normalizeOptions(
 
   const projectName = projectDirectory.replace(new RegExp('/', 'g'), '-');
   const fileName = getCaseAwareFileName({
-    fileName: options.simpleModuleName ? name : projectName,
+    fileName:
+      options.simpleName || options.simpleModuleName ? name : projectName,
```

Comment:
We don't support simpleModuleName here (its not in the schema.json), so we shouldn't need to check it.

```suggestion
      options.simpleName ? name : projectName,
```

Reply:
There is a `simpleModuleName` in the schema type, would you like it removed? I wasn't sure why it existed previously as its not in the schema.json

<img width="371" alt="image" src="https://user-images.githubusercontent.com/20795331/229230979-933e960d-dc9c-40cf-83d1-d3c444cab872.png">